### PR TITLE
Add the new type syntax support. Old type syntax doesn't work

### PR DESCRIPTION
**Pull request by roblabla**
PrismarineJS/minecraft-data#38

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ var mc = require('../')
   , MC_SERVER_JAR = process.env.MC_SERVER_JAR
   , SURVIVE_TIME = 10000
   , MC_SERVER_PATH = path.join(__dirname, 'server')
+  , getFieldInfo = require('../dist/utils').getFieldInfo
```
Comment (rom1504) [inline test/test.js]: this is wrong : tests should test only the exposed interface I think (as is done for the rest of the stuff used in test.js), so getFieldInfo and evalCondition should probably be exposed in index.js

**Comment by rom1504** [comment]
nice !

File: test/test.js (hunk 1 above)
Comment (roblabla) [inline test/test.js]: They aren't being tested, they're being used to set up the test. I really don't think exposing them is a good idea.

Comment (rom1504) [inline test/test.js]: ok

**Comment by rom1504** [comment]
bump node-minecraft-data to 0.6.0 and the tests should pass

**Pull request merged**